pkg/api/validation/schema.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/emicklei/go-restful/swagger"
 	"github.com/golang/glog"
-	"gopkg.in/v2/yaml"
+	"gopkg.in/yaml.v2"
```

Comment:
@thockin I seem to recall you wanting this change in a few places but I can't recall why?